TeamCode/src/main/java/org/firstinspires/ftc/teamcode/api/TriWheels.kt

```diff
@@ -9,30 +9,34 @@ import kotlin.math.*
  */
 object TriWheels : API() {
     // All 3 wheels on the robot
-    private lateinit var wheel1: DcMotor
-    private lateinit var wheel2: DcMotor
-    private lateinit var wheel3: DcMotor
+    lateinit var red: DcMotor
```

Comment:
what does lateinit mean?

Reply:
Usually uninitialized properties need to initialized by a class constructor. `lateinit` means that it doesn't need to be, but will raise an error if you try to access it before giving it a value.

[Kotlin Documentation](https://kotlinlang.org/docs/properties.html#late-initialized-properties-and-variables)

It is essentially the same as not initializing a variable in Java, but it gives better error handling.

```kotlin
class Foo {
    val bar: Int
    lateinit var baz: Int

    constructor() {
        // Must be initialized here or it will throw an error
        this.bar = 3

        // Will not throw an error if not initialized, but will if accessed before given a value
        // this.baz = ...
    }
}
```